File: cs50_week6/sentimental-mario-less/mario.py
def main():
    while True:
        try:
            n = int(input("Height: "))
            if n > 0 and n < 9:
                pyramid(n)
                break
        except ValueError:
            print("Not an int")


def pyramid(n):
    for row in range(n):
        hashtag = row + 1
        spaces = n - row - 1
        print(" " * spaces, end="")
        print("#" * hashtag)


main()

File: cs50_week1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int pyramidHeight = 0;
    do
    {
        //get user input
        pyramidHeight = get_int("How tall should the pyramid be? ");
    }
    while (pyramidHeight < 1 || pyramidHeight > 8);

    for (int row = 0; row < pyramidHeight; row++)
    {
        for (int spaces = 0; spaces < pyramidHeight - row - 1; spaces++)
        {
            printf(" ");
        }
        //first side
        for (int column = 0; column <= row; column++)
        {
            printf("#");
        }
        printf("  ");
        //second side
        for (int column = 0; column <= row; column++)
        {
            printf("#");
        }
        printf("\n");
    }
}